src/stepper/src/components/Step.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+
+import React, { Component } from 'react';
+import type { ChildrenArray } from 'react';
+
+import StepIcon from './StepIcon';
+import { StepWrapper, StepLabel } from './styled';
+
+type Props = {
+  active :boolean;
+  children :ChildrenArray<any>;
+  complete :boolean;
+  disabled ? :boolean;
+  index :number;
+  onClick :(e :SyntheticEvent<HTMLElement>) => void;
+};
+
+class Step extends Component<Props> {
+
+  static defaultProps = {
+    disabled: false
+  };
+
+  renderDefault = () => {
+    const {
+      active,
+      children,
+      complete,
+      index,
+    } = this.props;
+    return (
+      <>
+        <StepIcon
+            active={active}
+            complete={complete}
+            index={index} />
+        <StepLabel active={active}>{children}</StepLabel>
```

Comment:
Usually, there's a `for` or `forHtml` with labels. How is that handled in your implementation?

Reply:
This is actually rendering a `styled.span` It's just called `StepLabel` because that's it's purpose, rather than what it actually is. Open to renaming suggestions